log-viewer-cli/pom.xml

```diff
@@ -107,50 +100,46 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Dependency to `log-viewer-frontend` is not needed here, `log-viewer-frontend` will come to `log-viewer-cli` as a transitive dependency.

Reply:
no problem!